src/main/index.js

```diff
@@ -78,7 +78,7 @@ function createWindow() {
     icon: iconPath
   });
 
-  splashWindow.loadURL(getWindowPath(path.join(getStaticPath(), 'splash.html')));
+  splashWindow.loadURL(getWindowPath(path.join(getStaticPath(), 'splash.html'), 'splash.html'));
```

Comment:
Whoops.  I have a minor suggestion that I think will make this call less redundant.  Let's pass the path as the first parameter and the filename as the second parameter.  i.e.:

```js
function getWindowPath(productionPath, filename) {
     return isDev
       ? `http://localhost:${process.env.ELECTRON_WEBPACK_WDS_PORT}/${filename}`
       : url.format({ pathname: path.join(productionPath, filename), protocol: 'file:', slashes: true });
}

splashWindow.loadURL(getWindowPath(getStaticPath(), 'splash.html'));
mainWindow.loadURL(getWindowPath(__dirname, 'index.html'));
```